cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -95,25 +103,12 @@ def self.config(env)
             proxy: 'dashboard',
             headers: LANGUAGE_HEADER,
             cookies: whitelisted_cookies
-          },
-          {
-            path: %w(
-              /
-              /learn*
-              /congrats
-              /minecraft
-              /starwars
-              /playlab
-              /athletes
-            ),
-            headers: LANGUAGE_HEADER,
-            cookies: LANGUAGE_COOKIES,
           }
         ],
-        # Remaining Pegasus paths are English-only and don't require any extra headers or cookies.
+        # Remaining Pegasus paths are cached, and vary only on language and default cookies.
         default: {
-          headers: [],
-          cookies: 'none'
+          headers: LANGUAGE_HEADER,
+          cookies: DEFAULT_COOKIES
```

Comment:
Saw your other comment. There are only a few cases (/volunteer, /applab) where we have some I18n.t mixed with English. For /volunteer I think it's been up long enough that translations should be available for many languages so it's a benefit to turn localization on for this page. For /applab and any other cases I'll sort out whether to make them all English or all localized. Either way, I think this is a good change and these pages being in a mixed state shouldn't block merging in this PR.